From 5613cc86e7c9042b48b154180a4ad44fdac22b56 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 11 Apr 2013 15:49:00 +0800
Subject: [PATCH 187/281] ENGR00257933 BT: fix BT PAN profile not work issue

1. delete deprecated config "config_tether_upstream_regexs"
2. add bluetooth network support in config.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 .../overlay/frameworks/base/core/res/res/values/config.xml     | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
index f0ca849..e6ecf89 100755
--- a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
@@ -35,13 +35,6 @@
       <item>"wlan0"</item>
     </string-array>
 
-    <!-- Regex array of allowable upstream ifaces for tethering - for example if you want
-         tethering on a new interface called "foo2" add <item>"foo\\d"</item> to the array -->
-    <add-resource type="string" name="config_tether_upstream_regexs">
-       <item>ppp.*</item>
-       <item>wlan.*</item>
-    </add-resource>
-
     <!-- Array of ConnectivityManager.TYPE_xxxx values allowable for tethering -->
     <!-- Common options are [1, 4] for TYPE_WIFI and TYPE_MOBILE_DUN or
     <!== [0,1,5,7,9] for TYPE_MOBILE, TYPE_WIFI, TYPE_MOBILE_HIPRI, TYPE_BLUETOOTH and TYPE_ETHERNET -->
@@ -49,6 +42,7 @@
         <item>0</item>
         <item>1</item>
         <item>5</item>
+        <item>7</item>
         <item>9</item>
     </integer-array>
 
@@ -61,6 +55,7 @@
     <string-array translatable="false" name="radioAttributes">
         <item>"1,1"</item>
         <item>"0,1"</item>
+        <item>"7,1"</item>
         <item>"9,1"</item>
     </string-array>
 
@@ -80,6 +75,7 @@
         <item>"mobile_ims,11,0,3,60000,true"</item>
         <item>"mobile_cbs,12,0,3,60000,true"</item>
         <item>"wifi_p2p,13,1,0,-1,true"</item>
+        <item>"bluetooth,7,7,2,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
 
-- 
1.8.0

